docker/Dockerfile

```diff
@@ -181,6 +181,7 @@ COPY ./config /networks
 RUN mkdir /config
 
 RUN mkdir -p /node/db
+RUN mkdir -p /node/postgres
```

Comment:
What are we adding there? Is this folder necessary?

Reply:
yes it's needed otherwise we have permission issues